### DISCUSSION TITLE
2.15rc2

### DISCUSSION THREAD

**Discussion by gasman** [wagtail/wagtail#7662] [Announcements]
Changes from 2.15rc1:

* Revised deprecation timeline for database search backends; existing sites will continue to use the `wagtail.search.backends.db` backend unless they 'opt in' to the new `wagtail.search.backends.database` backend via `WAGTAILSEARCH_BACKENDS`; the new backend will become default in 2.17
* Fixed regression in image upload view causing users to be redirected to the single image upload form
* Updated edit actions on non-page models to show in logs as "Edited" rather than "Draft saved"
* Fixed data loss on TypedTableBlock when deleting then inserting columns
* Fixed TypedTableBlock styling to improve usability
* Fixed regression on InlinePanel causing delete / reorder buttons to fail to activate
* TypedTableBlock documentation fixes
* Updated translations

<hr /><em>This discussion was created from the release <a href='https://github.com/wagtail/wagtail/releases/tag/v2.15rc2'>2.15rc2</a>.</em>

**Comment by krzysztofjeziorny** [comment]
A minor error, "Add to collection" link in bulk actions navbar is not translated, to German at least in my use case.

(Should be called "Zur Sammlung hinzufügen", but don't quote me on this.)

![Screenshot 2021-11-03 at 15-55-18 Wagtail - Bilder](https://user-images.githubusercontent.com/872730/140084899-79617092-9cb2-4c9d-b34d-c9e7d450f2be.png)

**Comment by krzysztofjeziorny** [comment]
Hope here's the right place to note the bugs? Then I found one more - the bulk action confirmation form is not translated (again, German in my case).

![Screenshot 2021-11-03 at 16-08-38 Wagtail - Add 2 images to new collection](https://user-images.githubusercontent.com/872730/140087309-8a214f8c-d618-4b70-96b9-4d2302c57284.png)

**Comment by gasman** [comment]
Thanks @krzysztofjeziorny! Translations generally get added in the period between the release candidate and the final release, so release candidates will often be missing translations for newly-added strings. Looking at https://www.transifex.com/torchbox/wagtail/languages/, the German translation is still a little incomplete, so if you're able to help out by signing up to Transifex and contributing the missing ones, that would be much appreciated!

**Comment by krzysztofjeziorny** [comment]
Hey, thanks for the feedback! I'll take a look at the list, maybe I can help.
